packages/smooth_app/lib/generic_lib/widgets/app_bars/logged_in/logged_in_app_bar_body.dart

```diff
@@ -0,0 +1,130 @@
+import 'package:flutter/material.dart';
+import 'package:smooth_app/generic_lib/design_constants.dart';
+import 'package:smooth_app/generic_lib/widgets/app_bars/app_bar_background.dart';
+import 'package:smooth_app/generic_lib/widgets/app_bars/app_bar_constanst.dart';
+import 'package:smooth_app/generic_lib/widgets/app_bars/logged_in/app_bar_statistics_card.dart';
+import 'package:smooth_app/l10n/app_localizations.dart';
+import 'package:smooth_app/pages/preferences/lazy_counter.dart';
+import 'package:smooth_app/pages/preferences/user_preferences_page.dart';
+import 'package:smooth_app/query/paged_user_product_query.dart';
+import 'package:smooth_app/themes/theme_provider.dart';
+
+class LoggedInAppBarBody extends StatelessWidget {
+  const LoggedInAppBarBody({required this.userId, super.key});
+
+  final String userId;
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    final bool lightTheme = context.lightTheme();
+
+    return FlexibleSpaceBar(
+      collapseMode: CollapseMode.none,
+      background: Padding(
+        padding: const EdgeInsetsDirectional.only(bottom: SEARCH_BOTTOM_HEIGHT),
+        child: Stack(
+          children: <Widget>[
+            const AppBarBackground(height: LOGGED_IN_APP_BAR_EXPANDED_HEIGHT),
```

Comment:
AppBarBackground is just a CustomPaint.
You should remove the Stack here and : 
- Use the CustomPaint here directly (with a child)
- Give a child param to the AppBarBackground widget